tests/test_main.py

```diff
@@ -209,6 +216,28 @@ def test_no_init(tmp_dir):
     assert os.path.isdir("dvclive")
 
 
+def test_fail_on_conflict(tmp_dir, monkeypatch):
+    dvclive.init("some_dir")
+    monkeypatch.setenv(env.DVCLIVE_PATH, "logs")
+
+    with pytest.raises(ConfigMismatchError):
+        dvclive.log("m", 0.1)
+
+
+def test_dont_override_from_env(tmp_dir, monkeypatch):
```

Comment:
Not sure what are we testing here, might be related with my comment above about `from_env(existing_logger)` being outdated changes?

Reply:
Wrongly conducted rebase, removing.